modules/installation-and-upgrade/pages/container-deployment/generate_proxy_config.adoc

```diff
@@ -0,0 +1,90 @@
+// Add section on how to generated the proxy configuration files
+
+[[proxy-setup-containers-generate-config]]
+== Generate the Proxy Configuration
```

Comment:
This section have 3 sub-section that are all viable alternatives to generated the proxy configuration file. I didn't mentioned that they are alternatives, and users should use just one. Do you think we should make it more explicit?